include/ibamr/IBFEMethod.h

```diff
@@ -821,8 +821,19 @@ class IBFEMethod : public IBStrategy
     void putToDatabase(SAMRAI::tbox::Pointer<SAMRAI::tbox::Database> db) override;
 
     /*!
-     * Write the equation_systems data to a restart file in the specified directory.
-     */
+    *Restart files for the SAMRAI data is written using the built in RestartManager, but special restart files
+    *must be written for the IBFE objects. The function saves the solutions to the defined EquationSystems in 
+    *an xdr file for each FE part at each restart dump interval. An example snippet is included below to show 
+    *the distinct IBFE restart data saving step. The data will then be automatically read back into the system
+    *during the restart process. 
+    *
+    *    if (dump_restart_data && (iteration_num % restart_dump_interval == 0 || last_step))
+    *        {
+    *            RestartManager::getManager()->writeRestartFile(restart_dump_dirname, iteration_num);
+    *            ib_method_ops->writeFEDataToRestartFile(restart_dump_dirname, iteration_num);
+    *        }
+    *    }
+    */
```

Comment:
We should add a `@note` in the class documentation explaining how restart data works for this class.

Reply:
@drwells for clarification you want this @note to be added to the top of IBFEMethod.h?

Reply:
Yup: in the main documentation of the class.